packages/tools/viewer/src/viewer.ts

```diff
@@ -280,6 +306,16 @@ export class Viewer implements IDisposable {
      */
     public readonly onSkyboxBlurChanged = new Observable<void>();
 
+    /**
+     * Fired when the environment intensity changes.
+     */
+    public readonly onEnvironmentIntensityChanged = new Observable<void>();
+
+    /**
+     * Fired when the environment rotation changes.
+     */
+    public readonly onEnvironmentRotationChanged = new Observable<void>();
+

```

Comment:
Can we remove these? Part of the goal of bundling related properties together is to reduce the number of observables. So if we bundle, then we should just have the one `onEnvironmentConfigurationChanged` observable.